compiler/rustc_codegen_ssa/src/traits/backend.rs

```diff
@@ -50,8 +50,15 @@ pub trait CodegenBackend {
     /// - The second is like the first, but also includes unstable features.
     ///
     /// RUSTC_SPECIFIC_FEATURES should be skipped here, those are handled outside codegen.
-    fn target_features_cfg(&self, _sess: &Session) -> (Vec<Symbol>, Vec<Symbol>) {
-        (vec![], vec![])
+    fn target_config(&self, _sess: &Session) -> TargetConfig {
```

Comment:
Documentation of this function is outdated.

Also, why are the defaults `true`?

Reply:
Thanks for the documentation catch. Since the rollup is already testing, I'll fix this in a follow up.

`true` is the default since eventually we'll need to get there on (hopefully) all backends for the types to stabilize. It seemed better to make other backends explicitly ack that they don't support the type, than to quietly be skipping all f16/f128 tests and get a surprise when we go to stabilize.